README.md

```diff
@@ -21,26 +21,52 @@ If you don't happen to know what a carbon credit is:
 
 * Set pg and sequel.pg to use libpq
 
-       bundle config --local build.pg --with-opt-dir="/opt/homebrew/opt/libpq"
-       bundle config --local build.sequel_pg --with-opt-dir="/opt/homebrew/opt/libpq"
+        bundle config --local build.pg --with-opt-dir="/opt/homebrew/opt/libpq"
+        bundle config --local build.sequel_pg --with-opt-dir="/opt/homebrew/opt/libpq"
 
 * Install all Gems
 
-      bundle install
+        bundle install
+
+### The Datastore
+
+It's recommended to use a container for PostgreSQL or perhaps a 'cloud environment'.
+Export the following two environment variables with relevant PostgreSQL connection strings:
+
+        APP_TEST_DATABASE_URL
+        APP_DEV_DATABASE_URL
```

Comment:
The connection URL format for Postgres looks like:

`postgres://postgres@127.0.0.1:5432/patchApp_test`
`postgres://postgres@127.0.0.1:5432/patchApp_dev`

breaking this down it would be:

postges://{user_name}:{password}@{url}{port}/{database_name}.